addons/website_event_track/static/src/js/website_event_track.js

```diff
@@ -19,14 +19,20 @@ publicWidget.registry.websiteEventTrack = publicWidget.Widget.extend({
      */
     _onEventTrackSearchInput: function (ev) {
         ev.preventDefault();
-
         var text = $(ev.currentTarget).val();
-        var filter = _.str.sprintf(':containsLike(%s)', text);
-
-        $('#search_summary').removeClass('invisible');
         var $tracks = $('.event_track');
-        $('#search_number').text($tracks.filter(filter).length);
-        $tracks.removeClass('invisible').not(filter).addClass('invisible');
+        if (text)
+        {
+            var filter = _.str.sprintf(':containsLike(%s)', text);
```

Comment:
Could be great to explain this change as this may look like magic for external viewers of this commit. I would add a comment here and improve commit message accordingly.

Reply:
Added comment to explain the change